### PR TITLE
Edit Kubernetes Quick Start for clarity.

### PR DESCRIPTION
**Pull request by rcaballeromx**
This edit includes among other changes:

* Consistent use of markup, line length, and command formatting.
* Grammar, spelling, and other language fixes.
* Use of ordered lists for steps.
* New clear headings for installation options.

Signed-off-by: rcaballeromx <grca@google.com>

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit b37c6a4916fb0975de55592279c64d679b7a273b

https://app.netlify.com/sites/preliminary-istio/deploys/5b46536edd28ef7796a6bc78

**Comment by rcaballeromx** [comment]
This is my first pass at this document. However, the largest issue is the amount of information about requirements before the actual installation section. I want to create a separate PR where I create a separate document for the prerequisites and link to it. That change would provide a much better experience. @geeknoid Any thoughts?

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 1]
```diff
@@ -1,366 +1,463 @@
...
 
-To install Istio locally, install the latest version of
-[Minikube](https://kubernetes.io/docs/setup/minikube/) (version 0.28.0 or later).
+1. To install Istio locally, install the latest version of
+   [Minikube](https://kubernetes.io/docs/setup/minikube/), version **0.28.0 or
+   later**.
 
-Select a [VM driver](https://kubernetes.io/docs/tasks/tools/install-minikube/#install-a-hypervisor)
-and substitute the below `your_vm_driver_choice` with the installed VM driver.
+1. Select a [VM
```
Comment (sdake) [inline content/docs/setup/kubernetes/quick-start/index.md]: this should be one line - I suspect the cause of liting problem

Comment (rcaballeromx) [reply content/docs/setup/kubernetes/quick-start/index.md]: Thank you @sdake but that is not one of the issues the linter flags. There are several other links that are split like that due to line length but it does not flag any of them. 
My question there is, what should take precedence? Trying to keep the line length or keeping links on a single line?

**Comment by geeknoid** [comment]
You are not allowed to break an URL on two lines, so that's the cause of the warnings.

Also note that if you look in the netlify preview, you'll see that many of the code blocks aren't rendered correctly and display an error. This is because you changed the indent in code blocks. The indent of the content of {{< text >}} blocks should align with the {{< text >}} sequence.

**Comment by rcaballeromx** [comment]
@geeknoid I fixed the code blocks as you described but there are four that still won't render.
I also made sure all links were in a single line but the linter still complains. I will try removing the in-line markup within the link. I suspect that is what causes the flag. Do you know why the code blocks won't render even if they are on the same level as the tags?

**Comment by geeknoid** [comment]
Code blocks must always be indented by multiples of 4 spaces, that's probably what the issue is.

**Comment by rcaballeromx** [comment]
I finally got the linter to pass but there are two code blocks that are still not rendering correctly. Any suggestions on what the issue is?

**Pull request merged**